CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 11.0.0-rc.3 "merino-meadow" (2020-11-10)
+
```

Comment:
Should we add a line of text saying that there are no changes between rc.2 and rc.3?

Reply:
Ok I will add that in